playwright-e2e/data/tab_content/contested/express_case_not_qualify_gatekeeping_tab.ts

```diff
@@ -0,0 +1,7 @@
+export const expressDoesNotQualifyCaseGateKeepingTabData = [{
+    tabName: 'Gatekeeping and allocation',
+    tabContent: [
+      "Express Pilot Participation: Does not qualify",
+    ]
+  }
+]
```

Comment:
could you help me to apply it to `manage_express_case.spec.ts` as well please?

Reply:
Sure.  Will take a look.

Reply:
I've refactored express_case_gatekeeping_tabs.ts to be a bit clearer.  Look at where I imported that, same should work for you.